app/deepseg.cc

```diff
@@ -461,19 +493,30 @@ int main(int argc, char* argv[]) try {
 		}
 	}
 
+	// prevent use of both deprecated and current switches
+	if (setWorH && (capGeo || vidGeo)) {
+		showUsage = true;
+		fprintf(stderr, "Error: (DEPRECATED) -w/-h used in conjunction with --cg/--vg.\n");
```

Comment:
Maybe use `perror` instead?

Reply:
Only useful if `errno` has been set (which won't happen in this case).